arbitrage_opportunity/detector.py

```diff
@@ -85,6 +85,46 @@ def get_best_opportunity(tickers: List[ShortTicker]) -> Tuple[List[ShortTicker],
 
     return best_triplet, best_profit
 
+def get_best_opportunity(tickers: List[ShortTicker]) -> Tuple[List[ShortTicker], float]:
```

Comment:
Should we add a max cycle param here? Then we would be able to call this method inside `get_best_triangular_opportunity` with a max_cycle=3

Reply:
Will do.

Reply:
@Herklos what's a good default value? I am thinking 50?

Reply:
I don't have many ideas. 
I was thinking of a value much smaller than 50. For example, 5.

Reply:
Just changed it to 10, since our example yielded a great result where the cycle length is below 10 but greater than 5